pandas/tseries/tests/test_resample.py

```diff
@@ -1007,6 +1007,20 @@ def test_resample_timedelta_idempotency(self):
         expected = series
         assert_series_equal(result, expected)
 
+    def test_resample_timedelta_missing_values(self):
+        # GH 13223
+        index = pd.to_timedelta(['0s', pd.NaT, '2s'])
+        series = pd.Series([2, 3, 5], index=index)
+        result = series.resample('1s').mean()
+        expected = pd.Series([2, np.nan, 5], index=pd.timedelta_range(
+            start='0s', end='2s', freq='1s'))
+        assert_series_equal(result, expected)
+
+        # all NaT
+        index = pd.to_timedelta([pd.NaT, pd.NaT, pd.NaT])
+        series = pd.Series([2, 3, 5], index=index)
+        self.assertRaises(DataError, series.resample('1s').mean)
```

Comment:
This is probably the right idea (e.g. raising a nice message),

e.g. we do this for all-nan groups (which is unfriendly)

```
In [8]: s = Series([1,2,3],[np.nan,np.nan,np.nan])

In [9]: s.groupby(s.index).sum()
TypeError: unhashable type: 'Float64Index'
```